src/components/10-atoms/button/index.js

```diff
@@ -127,7 +127,7 @@ class AXAButton extends LitElement {
   disconnectedCallback() {
     super.disconnectedCallback();
 
-    if (this.isTypeSubmitOrReset) this.onclick = null;
+    if (this.isTypeSubmitOrReset) this.onClick = null;
```

Comment:
a `if` without `{}` is bad practise. Pls add them.

Reply:
I will🥶